README.md

```diff
@@ -30,6 +30,14 @@ To understand the structure, start with the file `hooli_data_eng/definitions.py`
 - *Retries* are enabled for both runs and assets, making the pipeline robust to occassional flakiness. See `hooli_data_eng/definitions.py` for examples of retries on jobs, and `hooli_data_eng/assets/marketing/__init__.py` for an example of a more complex retry policy on an asset including backoff and jitter. Flakiness is generated in `hooli_data_eng/resources/api.py`.
 - *Alerts* are enabled through Dagster Clould alert policies based on job tags. A custom alert is also specified to notify when assets with SLAs are later than expected. See `hooli_data_eng/assets/delayed_asset_alerts.py`. 
 
+## Assets spanning Multiple Code Locations
+
+- to run the `locations` dataset, uncomment out line 5-7 in [workspaces.yml](workspaces.yml)
+- you will also need to install the packages in the "sling" extra (e.g. `pip install -e ".[dev,sling]")`)
+- You'll need to obtain the credentials for AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
+- then run `dagster dev` and both `hooli_demo_assets` and `hooli_data_eng` code locations will load
+- NOTE: if you are running Sling locally, there is currently an error message if you already have a duckdb database set up the first time you run the Sling integration. You'll need to delete it and have the `location` asset be the the first thing you materialize.
```

Comment:
Did you happen to create a bug for this?  I vaguely remember running into it way back when.

Reply:
You mean on Sling's repo or here for us? Either way I didn't but can do

Reply:
I think we should raise it internally and then we can decide if it should go back to Sling too.

Reply:
done https://github.com/dagster-io/hooli-data-eng-pipelines/issues/79